### PR TITLE
swagger updated 

### PR DESCRIPTION
**Pull request by juliwhite**
NOTE: I updated the swagger-output file, so DO NOT run swagger. I couldn't figure it out how the swaggerAutogen.

### CODE REVIEW

**Review by carteraymond — APPROVED** [review]

File: src/controllers/categoryController.ts
Comment (BradyHodge) [inline src/controllers/categoryController.ts:1]: Why was this changed to a message?

Comment (carteraymond) [reply src/controllers/categoryController.ts:1]: the error message. some people consider them interchangeable or to mean both

Comment (BradyHodge) [reply src/controllers/categoryController.ts:1]: Yeah, it's not a big deal, Just curious

**Review by BradyHodge — APPROVED** [review]
